server/data/orchestrationApiClient.ts

```diff
@@ -249,9 +256,9 @@ export default class OrchestrationApiClient {
     })
   }
 
-  async getPrison(prisonCode: string): Promise<PrisonDto> {
+  async getPrison(prisonId: string): Promise<PrisonDto> {
     return this.restClient.get({
-      path: `/config/prisons/prison/${prisonCode}`,
+      path: `/config/prisons/prison/${prisonId}`,
```

Comment:
The prisonId / prisonCode bring immense frustration from my point of view, probably better if we do decide to always use the same one ignoring the requirement of the orchestration

Reply:
It's pretty consistent in this repo which is why I changed this. Admin UI is all over the place with code vs id 😂